src/index.tsx

```diff
@@ -64,22 +64,20 @@ const initializeCourseViewer = (): void => {
   }
 
   const courseArray = Object.keys(courseData);
-  const courseItems = courseArray.map((name) => (
-    <Course key={name} authLevel={authLevel} course={courseData[name]} />
+  const courseItems = courseArray.map((name, index) => (
```

Comment:
This actually isn’t good for performance because the index can change when iterating and then react will have to update more components

Reply:
Hmm yeah you're right. I think using the course ID is better than the name though. Using the name seems kinda risky.

Reply:
Although, using the course ID means we need to make an array access which will impact performance. I think I'll just leave it with name.